packages/terra-site/webpack.config.js

```diff
@@ -45,6 +51,7 @@ module.exports = {
                     'iOS >= 8',
                   ],
                 }),
+                CustomProperties(),
               ];
```

Comment:
I recommend we add it, i dig into the code and the plugin returns if the `[dir]` attribute is detected https://github.com/vkalinichev/postcss-rtl/blob/master/src/index.js#L32